src/api/media_engine/mod.rs

```diff
@@ -280,6 +283,17 @@ impl MediaEngine {
                 payload_type: 125,
                 ..Default::default()
             },
+            RTCRtpCodecParameters {
```

Comment:
I don't think this should be registered as a default codec. At least, at first it should be opt in via `register_codec`

Reply:
I can remove it, but I added it because chrome has it by default now (as of chrome 90 https://www.macrumors.com/2021/04/15/chrome-90-https-av1-codec-video-conferencing/) 

<img width="590" alt="Screen Shot 2022-05-24 at 11 23 51 AM" src="https://user-images.githubusercontent.com/227376/170072999-9e57a514-5bb8-411a-a562-a678ad27f2d9.png">

Reply:
I think we leave it out for now. Can you remove this addition and folks can use `register_codec` and I'll merge it.l